bindings/open3D/aditof_open3d.h

```diff
@@ -0,0 +1,64 @@
+#ifndef ADITOF_OPENCV_H
+#define ADITOF_OPENCV_H
+
+#include <Open3D/Open3D.h>
+#include <aditof/frame.h>
+#include <aditof/status_definitions.h>
+
+using namespace open3d;
+
+namespace aditof {
+aditof::Status fromFrameToDepthImg(aditof::Frame &frame, int camera_range,
+                                   geometry::Image &image) {
+    aditof::FrameDetails frameDetails;
```

Comment:
This function is defined inside the aditof namespace, no need to add the namespace to FrameDetails. Check the rest of the file for more issues like this.

Reply:
solved. thanks